playground/DurableTask/DurableTask.Scheduler.AppHost/Program.cs

```diff
@@ -0,0 +1,23 @@
+using Aspire.Hosting.Azure;
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var scheduler =
+    builder.AddDurableTaskScheduler("scheduler")
+           .RunAsEmulator(
+                options =>
+                {
+                    options.WithDynamicTaskHubs();
```

Comment:
Why is this explicit call needed?

Reply:
Just as a demonstration of configuring options on the emulator.  (Not every Aspire application will want to use dynamic task hub names.)